views/addGroup.njk

```diff
@@ -42,6 +42,7 @@
           {{ govukButton({
             text: "Confirm",
             type: "submit",
+            classes: "govuk-!-margin-right-1",
```

Comment:
found in govuk design system that their buttons were better spaced because they had added this to all of them